Makefile

```diff
@@ -1,22 +1,39 @@
 TARGETS := e2e suite
+VERSION ?= $(shell ./version.sh)
+GIT_COMMIT := $(shell git rev-parse HEAD)
+VERSIONFILE := version.go
+# docker doesn't allow "+" in image tags: https://github.com/docker/distribution/issues/1201
+DOCKER_VERSION ?= $(subst +,-,$(VERSION))
+TAG ?= latest
```

Comment:
In order to discourage use of named tags, I considered removing this variable & related infrastructure.  I believe you use this regularly @a-palchikov (particularly the `dima` tag) which is why I chose to leave it in place presently.